private/bufpkg/bufcheck/bufbreaking/internal/bufbreakingcheck/util.go

```diff
@@ -22,65 +22,65 @@ import (
 	"github.com/bufbuild/buf/private/bufpkg/bufanalysis"
 	"github.com/bufbuild/buf/private/bufpkg/bufcheck/internal"
 	"github.com/bufbuild/buf/private/bufpkg/bufprotosource"
-	"google.golang.org/protobuf/types/descriptorpb"
+	"google.golang.org/protobuf/reflect/protoreflect"
 )
 
 var (
 	// https://developers.google.com/protocol-buffers/docs/proto3#updating
-	fieldDescriptorProtoTypeToWireCompatiblityGroup = map[descriptorpb.FieldDescriptorProto_Type]int{
-		descriptorpb.FieldDescriptorProto_TYPE_INT32:  1,
-		descriptorpb.FieldDescriptorProto_TYPE_INT64:  1,
-		descriptorpb.FieldDescriptorProto_TYPE_UINT32: 1,
-		descriptorpb.FieldDescriptorProto_TYPE_UINT64: 1,
-		descriptorpb.FieldDescriptorProto_TYPE_BOOL:   1,
-		descriptorpb.FieldDescriptorProto_TYPE_SINT32: 2,
-		descriptorpb.FieldDescriptorProto_TYPE_SINT64: 2,
+	fieldDescriptorProtoTypeToWireCompatiblityGroup = map[protoreflect.Kind]int{
```

Comment:
The name of this map should change, right?

Reply:
Ah, yes. Done. Changed `fieldDescriptorProtoType` to `fieldKind` in this map name and the other one below.